lib/views/walks/walk_results_list.dart

```diff
@@ -27,14 +27,15 @@ class WalkResultsList extends StatelessWidget {
             if (i == 0) {
               return ListHeader(_defineTopHeader());
             }
-            if (i == 6) {
-              return const ListHeader("Autres Points");
-            }
-            if (i < 6) {
-              i = i - 1;
-            } else {
-              i = i - 2;
-            }
+            // if (i == 6) {
```

Comment:
If we don't separate walks with trips anymore, maybe we should then remove this whole custom logic and just display the whole list after the header.